static/app/views/replays/replayTable/headerCell.tsx

```diff
@@ -25,9 +25,33 @@ function HeaderCell({column, sort}: Props) {
     case ReplayColumn.BROWSER:
       return <SortableHeader sort={sort} fieldName="browser.name" label={t('Browser')} />;
 
+    case ReplayColumn.COUNT_DEAD_CLICKS:
+      return (
+        <SortableHeader
+          sort={sort}
+          fieldName="count_dead_clicks"
+          label={t('Dead Clicks')}
+          tooltip={t(
+            'Dead clicks are slow clicks that are too slow and cause a timeout.'
+          )}
```

Comment:
cc @mydea @billyvg to clarify the explanation here and below.

We also need to followup with the ui implementation too, it's not doing anything to distinguish multi-click from rage-click:

https://github.com/getsentry/sentry/blob/master/static/app/utils/replays/frame.tsx#L130-L135

Reply:
Maybe:

> Dead clicks are slow clicks that do not result in any page activity.

Or something like this? Timeout is a bit ambiguous, I'd say.